mycelite/src/config.rs

```diff
@@ -432,19 +290,19 @@ unsafe extern "C" fn x_update(
     let vtab = VTab::as_mut(vtab);
     let config = ConfigRegistry::new().get(vtab.database_path.as_str());
     let mut config = config.lock().unwrap();
-    match UpdateType::from((argc, value)) {
+    match UpdateType::from((argc, value, SQLITE3_API)) {
         UpdateType::Delete {
             row_id: SqliteValue::I64(row_id),
         } => config.delete(row_id as usize),
-        UpdateType::Update { columns, .. } => match (columns.get(0), columns.get(1)) {
+        UpdateType::Update { mut columns, .. } => match (columns.next(), columns.next()) {
```

Comment:
Out of curiosity, does `next() + next()` really look clearer than `get(0) + get(1)`? I am fine with either tho.

Reply:
previously it was vec, but since libsqlite-sys is no-std, we can't have Vec, instead we can pass iterator and get access to elements through Iterator::next()

Reply:
Ah I see, my assumption was clippy complaining or some such. Makes perfect sense, thanks for clarifying